parquet-column/src/main/java/org/apache/parquet/example/data/simple/convert/GroupRecordConverter.java

```diff
@@ -46,7 +47,8 @@ public void end() {
 
   @Override
   public Group getCurrentRecord() {
-    return root.getCurrentRecord();
+    SimpleGroup simpleGroup = (SimpleGroup) root.getCurrentRecord();
+    return simpleGroup.isNull() ? null : simpleGroup;
```

Comment:
Good point. This seems like some leftover code so I removed the relevant commit, which also removes the `isNull()` method from `SimpleGroup` (related to your comment above).